### ISSUE TITLE
Non-scalar `get` doesn't work?

### ISSUE THREAD

**Issue by The0x539** [uiua-lang/uiua#619]
This example from the website:
```
map 1_2_3 4_5_6
get 1_3 .
```
works as expected for me in the online interpreter, but fails for me locally with the message:
```
Error: Key not found in map
  at 2:1
2 | get 1_3 .
    ───
```
I've tried with both 0.13.0 and 0.14.0-dev.5, and I cannot for the life of me figure out what I might be doing wrong, given that I've literally copy-pasted the code that works in browser.

**Comment by kaikalii** [comment]
This may require newer that `0.14.0-dev.5`

**Comment by kaikalii** [comment]
I've just published `0.14.0-dev.6`, which should have this.
